clang/test/SemaSYCL/registered-kernel-names.cpp

```diff
@@ -0,0 +1,87 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -pedantic %s
```

Comment:
Is pedantic flag somehow important in this test?
```suggestion
// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
```